Frends.OAuth.CreateJWTToken/Frends.OAuth.CreateJWTToken.Tests/UnitTests.cs

```diff
@@ -0,0 +1,137 @@
+using Frends.OAuth.CreateJWTToken.Definitions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Frends.OAuth.CreateJWTToken.Tests;
+
+[TestClass]
+public class UnitTests
+{
+    private static readonly string privateKey = @"-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I think this should be moved to a file and read from there. Now the key is taking quite a lot of lines.